app/pathogen/arbovirus/analyze/recharts.tsx

```diff
@@ -25,6 +25,15 @@ import clsx from "clsx";
 
 //Study count by pathogen and antibody type
 
+export enum ShortformArbovirus {
```

Comment:
I'd like to move us towards an enum. The big advantage is that I can `Object.values` it to get every enum value which I make use of in this PR. The codegen library I want to use here also uses enums instead of `'X' | 'Y' | 'Z'` so don't worry about that :)